packages/integrations/vue/test/toTsx.test.cts

```diff
@@ -0,0 +1,45 @@
+import { toTSX } from '../src/editor.cts';
+import {  describe, it } from 'node:test';
+import { strictEqual } from 'node:assert'
```

Comment:
Our tests uses `import * as assert from 'node:assert/strict';`, I think we should refactor to that for consistency.

Reply:
Thanks for your suggestion, I have changed it.